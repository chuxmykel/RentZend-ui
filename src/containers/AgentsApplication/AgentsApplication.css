.application {
  display: flex;
  min-height: calc(100vh - 64px);
  justify-content: center;
  align-items: center;
}

.application-form-card {
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 50px;
}

.application-form-card form {
  width: 400px;
}