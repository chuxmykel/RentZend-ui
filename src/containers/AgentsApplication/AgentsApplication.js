import React from 'react';
import ApplicationForm from '../../components/Forms/ApplicationForm';
import './AgentsApplication.css';

const AgentsApplication = () => {
  return (
    <div className="application">
      <ApplicationForm />
    </div>
  );
}

export default AgentsApplication;
