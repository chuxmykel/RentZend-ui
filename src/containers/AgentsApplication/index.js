import AgentsApplication from './AgentsApplication';

export default AgentsApplication;
