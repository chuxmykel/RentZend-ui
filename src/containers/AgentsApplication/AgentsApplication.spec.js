import React from 'react';
import { shallow } from 'enzyme';
import AgentsApplication from './AgentsApplication';

describe('<AgentsApplication /> with no props', () => {
  const container = shallow(<AgentsApplication />);
  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

  it('should have the main div', () => {
    expect(container.find('div').length).toEqual(1);
  });
});
