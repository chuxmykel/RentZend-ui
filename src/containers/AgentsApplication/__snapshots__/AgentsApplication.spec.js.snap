// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AgentsApplication /> with no props should match the snapshot 1`] = `"<div class=\\"application\\"><div class=\\"MuiPaper-root MuiCard-root application-form-card MuiPaper-elevation8 MuiPaper-rounded\\"><form><div class=\\"MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined Mui-required Mui-required\\" data-shrink=\\"false\\" for=\\"name\\" id=\\"name-label\\">full name<span class=\\"MuiFormLabel-asterisk MuiInputLabel-asterisk\\"> *</span></label><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense\\"><input type=\\"text\\" aria-invalid=\\"false\\" id=\\"name\\" name=\\"name\\" required=\\"\\" value=\\"\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense\\"/><fieldset aria-hidden=\\"true\\" class=\\"PrivateNotchedOutline-root-89 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legendLabelled-91\\"><span>full name *</span></legend></fieldset></div></div><div class=\\"MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined Mui-required Mui-required\\" data-shrink=\\"false\\" for=\\"email\\" id=\\"email-label\\">email<span class=\\"MuiFormLabel-asterisk MuiInputLabel-asterisk\\"> *</span></label><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense\\"><input type=\\"email\\" aria-invalid=\\"false\\" id=\\"email\\" name=\\"email\\" required=\\"\\" value=\\"\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense\\"/><fieldset aria-hidden=\\"true\\" class=\\"PrivateNotchedOutline-root-89 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legendLabelled-91\\"><span>email *</span></legend></fieldset></div></div><div class=\\"MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined Mui-required Mui-required\\" data-shrink=\\"false\\" for=\\"phoneNumber\\" id=\\"phoneNumber-label\\">phone number<span class=\\"MuiFormLabel-asterisk MuiInputLabel-asterisk\\"> *</span></label><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense\\"><input type=\\"text\\" aria-invalid=\\"false\\" id=\\"phoneNumber\\" name=\\"phoneNumber\\" required=\\"\\" value=\\"\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense\\"/><fieldset aria-hidden=\\"true\\" class=\\"PrivateNotchedOutline-root-89 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legendLabelled-91\\"><span>phone number *</span></legend></fieldset></div></div><div class=\\"MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined Mui-required Mui-required\\" data-shrink=\\"false\\" for=\\"zipCode\\" id=\\"zipCode-label\\">zipCode<span class=\\"MuiFormLabel-asterisk MuiInputLabel-asterisk\\"> *</span></label><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense\\"><input type=\\"text\\" aria-invalid=\\"false\\" id=\\"zipCode\\" name=\\"zipCode\\" required=\\"\\" value=\\"\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense\\"/><fieldset aria-hidden=\\"true\\" class=\\"PrivateNotchedOutline-root-89 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legendLabelled-91\\"><span>zipCode *</span></legend></fieldset></div></div><input type=\\"file\\" accept=\\"image/*\\" class=\\"makeStyles-input-1\\" id=\\"passport\\" name=\\"passport\\"/><label for=\\"passport\\"><span class=\\"MuiButtonBase-root MuiIconButton-root MuiIconButton-colorPrimary\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\" aria-label=\\"upload picture\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\"><circle cx=\\"12\\" cy=\\"12\\" r=\\"3.2\\"></circle><path d=\\"M9 2L7.17 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2h-3.17L15 2H9zm3 15c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z\\"></path></svg><p class=\\"MuiTypography-root MuiTypography-body1\\" style=\\"margin-left:10px\\">Click to upload passport photograph</p></span></span></label><div class=\\"MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined Mui-required Mui-required\\" data-shrink=\\"false\\" for=\\"address\\" id=\\"address-label\\">address<span class=\\"MuiFormLabel-asterisk MuiInputLabel-asterisk\\"> *</span></label><div class=\\"MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline MuiInputBase-marginDense MuiOutlinedInput-marginDense\\"><textarea aria-invalid=\\"false\\" id=\\"address\\" name=\\"address\\" required=\\"\\" rows=\\"4\\" class=\\"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense\\"></textarea><fieldset aria-hidden=\\"true\\" class=\\"PrivateNotchedOutline-root-89 MuiOutlinedInput-notchedOutline\\"><legend class=\\"PrivateNotchedOutline-legendLabelled-91\\"><span>address *</span></legend></fieldset></div></div><button class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-fullWidth\\" tabindex=\\"0\\" type=\\"submit\\" style=\\"margin-top:20px\\"><span class=\\"MuiButton-label\\">Submit</span></button></form></div></div>"`;
